feature/ledgermanual/src/main/java/com/moneymong/moneymong/ledgermanual/view/LedgerManualCategoryBottomSheet.kt

```diff
@@ -0,0 +1,301 @@
+                style = Body3,
+                color = Blue04,
+                text = "추가",
+            )
+        }
+        Spacer(modifier = Modifier.height(4.dp))
+        Text(
+            style = Body2,
+            color = Gray05,
+            text = "원하는 카테고리를 마음대로 만들 수 있어요",
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        FlowRow(
+            horizontalArrangement = Arrangement.spacedBy(12.dp),
+        ) {
+            categories.forEach {
+                MDSOutlineTag(
+                    text = it,
+                    iconResource = R.drawable.ic_close_default,
+                    onClick = {},
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun LedgerManualCategoryCreateBottomSheetContent(
+    modifier: Modifier = Modifier,
+    textFieldValue: TextFieldValue,
+    isSystemCategoryError: Boolean,
+    categories: List<String>, // TODO API response
+    onValueChange: (TextFieldValue) -> Unit,
+    onClickRegister: () -> Unit,
+    onPrev: () -> Unit,
+) {
+    val maxCount = 10
+    var isFilled by remember { mutableStateOf(false) }
+    val focusRequester = remember { FocusRequester() }
+    val keyboard = LocalSoftwareKeyboardController.current
+    val isExists = categories.contains(textFieldValue.text)
+    val helperText by remember(isSystemCategoryError, isExists) {
+        derivedStateOf {
+            when {
+                isSystemCategoryError -> "사용할 수 없는 카테고리 이름이에요"
+                isExists -> "이미 있는 카테고리에요"
+                else -> ""
+            }
+        }
+    }
+
+    BackHandler {
+        keyboard?.hide()
+        onPrev()
+    }
+
+    LaunchedEffect(Unit) {
+        focusRequester.requestFocus()
+    }
+
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(White)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(White)
+                .padding(horizontal = MMHorizontalSpacing, vertical = 20.dp),
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.spacedBy(8.dp),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                Icon(
+                    modifier = Modifier
+                        .size(24.dp)
+                        .noRippleClickable {
+                            keyboard?.hide()
+                            onPrev()
+                        },
+                    painter = painterResource(R.drawable.ic_chevron_left),
+                    contentDescription = null,
+                    tint = Gray07,
+                )
+                Text(
+                    text = "카테고리 생성",
+                    style = Heading1,
+                    color = Black,
+                )
+            }
+            Spacer(modifier = Modifier.height(36.dp))
```

Comment:
디자인상 16으로 확인됩니다!(`MDSTextField`가 `Title` 높이를 항상 차지하고 있습니다!)

```suggestion
            Spacer(modifier = Modifier.height(16.dp))
```

Reply:
디테일 굿입니다 👍